src/org/traccar/database/DataManager.java

```diff
@@ -123,6 +203,56 @@ private String getQuery(String key) {
         return query;
     }
 
+    public String getQuery(String action, Class<?> clazz) {
+        return getQuery(action, clazz, false);
+    }
+
+    public String getQuery(String action, Class<?> clazz, boolean extended) {
+        String queryName;
+        if (action.equals(ACTION_SELECT_ALL)) {
+            queryName = "database.select" + clazz.getSimpleName() + "s";
+        } else {
+            queryName = "database." + action.toLowerCase() + clazz.getSimpleName() + (extended ? "Extended" : "");
```

Comment:
I would split the `extended` check into a new line with a regular `if` statement.